packages/documentation/pages/usage/components/field-inline-edit.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+	<div>
+		<ComponentInfo v-bind="{ component }" />
+
+		<div class="element-example white">
+			<KtForm v-model="formValue">
+				<!-- v-model="confirmedValue" -->
```

Comment:
the commented code in this file is all to swap out the field to not be part of a form if needed